.travis.yml

```diff
@@ -150,16 +149,6 @@ install:
           export QT_API=pyqt;
       fi
 
-    # Reinstalling the required numpy version as it may be downgraded.
-    # TODO It may be removed when/if Sherpa becomes np version agnostic
-    # (maybe as soon as v4.8). See discussion in
-    # https://travis-ci.org/gammapy/gammapy/jobs/99867950#L639
-    - if $SHERPA; then
-          conda install --yes --channel astropy numpy=$NUMPY_VERSION iminuit;
-          conda install --yes --no-pin --channel https://conda.anaconda.org/cxc/channel/dev sherpa;
```

Comment:
Thanks. I'll try not to mess up things.

As for the sherpa dev, we would need a fortran compiler, too. Maybe later in the evening I can figure out how difficult to get one for that build (or will see whether there will be weekly conda dev packages).